### PR TITLE
[Doc] add 1.2.9 flink connector rn

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: docs/en/ecosystem_release/flink_connector.md [hunk 1]
```diff
@@ -28,18 +28,33 @@ displayed_sidebar: "English"
 
 | Connector | Flink                    | StarRocks     | Java | Scala     |
 | --------- | ------------------------ | ------------- | ---- | --------- |
+| 1.2.9     | | |     |  |
```
Comment (hellolilyliuyi) [inline docs/en/ecosystem_release/flink_connector.md:31]: 1.2.9 flinkconnector对其他应用版本支持度辛苦填一下

File: docs/en/ecosystem_release/flink_connector.md [hunk 2]
```diff
@@ -28,18 +28,33 @@ displayed_sidebar: "English"
...
+
+**Features**
+
+- Implement the new sink API introduced in [FLP-191](https://cwiki.apache.org/confluence/display/FLINK/FLIP-191%3A+Extend+unified+Sink+interface+to+support+small+file+compaction) to support [Flink CDC 3.0](https://github.com/ververica/flink-cdc-connectors/issues/2600). [#301](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/301)
+- Support Flink 1.18. [#305](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/305)
+
+**Bug Fixes**
+
+- Fix misleading thread name and log. [#290](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/290)
+- Fix wrong configurations when writing to multiple tables. [#298](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/298)
```
Comment (hellolilyliuyi) [inline docs/en/ecosystem_release/flink_connector.md:55]: 这个是stream loadsdk的配置？

File: docs/zh/loading/Flink-connector-starrocks.md [hunk 3]
```diff
@@ -202,6 +203,18 @@ Flink connector 提供以下指标来监控导入情况。
 | totalFlushFailedTimes    | Counter | flush 数据的失败次数。                                   |
 | totalFilteredRows        | Counter | 已过滤的行数，这些行数也包含在 totalFlushRows 中。 |
 
+### Flink CDC 同步（支持 schema change）
+
+Flink 发布的 [Flink CDC 3.0 框架](ttps://github.com/ververica/flink-cdc-connectors/releases)可以轻松地从 CDC 数据源（如 MySQL、Kafka）到 StarRocks 构建[流式 ELT 管道](https://ververica.github.io/flink-cdc-connectors/master/content/overview/cdc-pipeline.html) 。该管道能够将整个数据库、合并分片表以及来自源端的 schema change 同步到 StarRocks。
```
Comment (banmoy) [inline docs/zh/loading/Flink-connector-starrocks.md:208]: Flink CDC 3.0现在还不是Flink项目下的，不能这么说，就说最新发布的就可以可以了

File: docs/en/ecosystem_release/flink_connector.md [hunk 4]
```diff
@@ -28,18 +28,34 @@ displayed_sidebar: "English"
...
 
-## **Release note**
+## Release note
 
 ### 1.2
 
-**1.2.8**
+#### 1.2.9
+
+This release includes some features and bug fixes. The notable change is that the Flink connector is integrated with [Flink CDC 3.0](https://ververica.github.io/flink-cdc-connectors/master/content/overview/cdc-pipeline.html) to easily build a streaming ELT pipeline from CDC sources (such as MySQL, Kafka) to StarRocks. You can see [Flink CDC Synchronization](../loading/Flink-connector-starrocks.md#flink-cdc-synchronization-schema-change-supported) for details.
```
Comment (amber-create) [inline docs/en/ecosystem_release/flink_connector.md:45]: such as MySQL, Kafka -> such as MySQL and Kafka

File: docs/en/ecosystem_release/flink_connector.md [hunk 5]
```diff
@@ -28,18 +28,34 @@ displayed_sidebar: "English"
...
+
+**Features**
+
+- Implement catalog to support Flink CDC 3.0. [#295](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/295)
+- Implement new sink api in [FLP-191](https://cwiki.apache.org/confluence/display/FLINK/FLIP-191%3A+Extend+unified+Sink+interface+to+support+small+file+compaction) to support Flink CDC 3.0. [#301](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/301)
```
Comment (amber-create) [inline docs/en/ecosystem_release/flink_connector.md:50]: api -> API

File: docs/en/ecosystem_release/flink_connector.md [hunk 6]
```diff
@@ -28,18 +28,34 @@ displayed_sidebar: "English"
...
+- Support Flink 1.18. [#305](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/305)
+
+**Bug Fixes**
+
+- Fix misleading thread name and log. [#290](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/290)
+- Fix wrong stream-load-sdk configurations when writing to multiple tables. [#298](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/298)
```
Comment (amber-create) [inline docs/en/ecosystem_release/flink_connector.md:56]: when -> used for?

File: docs/zh/ecosystem_release/flink_connector.md [hunk 7]
```diff
@@ -41,7 +42,22 @@ displayed_sidebar: "Chinese"
 
 ### 1.2
 
-**1.2.8**
+#### 1.2.9
+
+本版本发布包含如下新增特性和问题修复。值得注意的变化是，Flink connector 已与 Flink CDC 3.0集成，可轻松地从 CDC 数据源（如 MySQL、Kafka）构建流式 ELT 管道到 StarRocks。更多信息，您参见 [Flink CDC 同步（支持 schema change）](../loading/Flink-connector-starrocks.md#flink-cdc同步支持-schema-change)。
```
Comment (amber-create) [inline docs/zh/ecosystem_release/flink_connector.md:47]: image
加个空格

File: docs/zh/loading/Flink-connector-starrocks.md [hunk 8]
```diff
@@ -107,7 +108,7 @@ Flink connector JAR 文件的命名格式如下：
 | sink.buffer-flush.max-rows        | No       | 500000        | 积攒在内存的数据条数，达到该阈值后数据通过 Stream Load 一次性导入 StarRocks。取值范围：[64000, 5000000]。该参数只在 `sink.version` 为 `V1`，`sink.semantic` 为 `at-least-once` 才会生效。 |
 | sink.buffer-flush.interval-ms     | No       | 300000        | 数据发送的间隔，用于控制数据写入 StarRocks 的延迟，取值范围：[1000, 3600000]。该参数只在 `sink.semantic` 为 `at-least-once`才会生效。 |
 | sink.max-retries                  | No       | 3             | Stream Load 失败后的重试次数。超过该数量上限，则数据导入任务报错。取值范围：[0, 10]。该参数只在 `sink.version` 为 `V1` 才会生效。 |
-| sink.connect.timeout-ms           | No       | 1000          | 与 FE 建立 HTTP 连接的超时时间。取值范围：[100, 60000]。     |
+| sink.connect.timeout-ms           | No       |  30000            | 与 FE 建立 HTTP 连接的超时时间。取值范围：[100, 60000]。  Flink connector v1.2.9 之前，默认值为1000。  |
```
Comment (amber-create) [inline docs/zh/loading/Flink-connector-starrocks.md:111]: image
加个空格，1000 加 code format

File: docs/en/loading/Flink-connector-starrocks.md [hunk 9]
```diff
@@ -104,7 +105,7 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
...
+| sink.connect.timeout-ms           | No           | 30000              | The timeout for establishing HTTP connection. Valid values: 100 to 60000. Unit: ms.  Before Flink connector v1.2.9, the default value is 1000.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (amber-create) [inline docs/en/loading/Flink-connector-starrocks.md:108]: image
Remove extra space preceding "Before", and wrap "1000" in code format.

File: docs/en/loading/Flink-connector-starrocks.md [hunk 10]
```diff
@@ -219,6 +220,20 @@ The Flink connector provides the following metrics to monitor loading.
 | totalFlushFailedTimes    | counter | number of times that the data fails to be flushed.                  |
 | totalFilteredRows        | counter | number of rows filtered, which is also included in totalFlushRows.    |
 
+### Flink CDC Synchronization (schema change supported)
+
+[Flink CDC 3.0](https://github.com/ververica/flink-cdc-connectors/releases) framework can be used
+to easily [build a streaming ELT pipeline from CDC sources](https://ververica.github.io/flink-cdc-connectors/master/content/overview/cdc-pipeline.html) (such as MySQL, Kafka) to StarRocks. The pipeline can synchronize whole
```
Comment (amber-create) [inline docs/en/loading/Flink-connector-starrocks.md:226]: such as MySQL, Kafka -> such as MySQL and Kafka

Comment (amber-create) [inline docs/en/loading/Flink-connector-starrocks.md:226]: image
Why does the line break at "whole"?

File: docs/en/loading/Flink-connector-starrocks.md [hunk 11]
```diff
@@ -219,6 +220,20 @@ The Flink connector provides the following metrics to monitor loading.
...
+database, merged sharding tables, and schema changes from sources to StarRocks.
+
+Since v1.2.9, the Flink connector for StarRocks is integrated into this framework as [StarRocks Pipeline Connector](https://ververica.github.io/flink-cdc-connectors/master/content/pipelines/starrocks-pipeline.html). The StarRocks Pipeline Connector supports:
+
+- Create database/table automatically
```
Comment (amber-create) [inline docs/en/loading/Flink-connector-starrocks.md:231]: -> Automatically create databases and tables

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — DISMISSED** [review]

**Review by EsoragotoSpirit — DISMISSED** [review]

**Review by evelynzhaojie — DISMISSED** [review]

File: docs/zh/loading/Flink-connector-starrocks.md [hunk 12]
```diff
@@ -202,6 +203,18 @@ Flink connector 提供以下指标来监控导入情况。
 | totalFlushFailedTimes    | Counter | flush 数据的失败次数。                                   |
 | totalFilteredRows        | Counter | 已过滤的行数，这些行数也包含在 totalFlushRows 中。 |
 
+### Flink CDC 同步（支持 schema change）
+
+[Flink CDC 3.0 框架](https://github.com/ververica/flink-cdc-connectors/releases)可以轻松地从 CDC 数据源（如 MySQL、Kafka）到 StarRocks 构建[流式 ELT 管道](https://ververica.github.io/flink-cdc-connectors/master/content/overview/cdc-pipeline.html)。该管道能够将整个数据库、分库分表以及来自源端的 schema change 同步到 StarRocks。
+
+自 v1.2.9 起，StarRocks 提供的 Flink connector 已经集成至该框架中，并且被命名为 [StarRocks Pipeline Connector]((https://ververica.github.io/flink-cdc-connectors/master/content/pipelines/starrocks-pipeline.html)。StarRocks Pipeline Connector 支持：
```
Comment (evelynzhaojie) [inline docs/zh/loading/Flink-connector-starrocks.md:210]: 前面多括号

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Pull request merged**